modules/swagger-codegen/src/main/resources/php/model_generic.mustache

```diff
@@ -154,14 +158,14 @@ class {{classname}} {{#parentSchema}}extends {{{parent}}} {{/parentSchema}}{{^pa
      * @param mixed[] $data Associated array of property values
      *                      initializing the model
      */
-    public function __construct(array $data = null)
+    public function __construct(array $data = [])
     {
         {{#parentSchema}}
         parent::__construct($data);
 
         {{/parentSchema}}
         {{#vars}}
-        $this->container['{{name}}'] = isset($data['{{name}}']) ? $data['{{name}}'] : {{#defaultValue}}{{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}null{{/defaultValue}};
+        array_key_exists('{{name}}', $data) && $this->container['{{name}}'] = $data['{{name}}'];
```

Comment:
I think it would be `==` instead of `=`

Reply:
It's an assignation that is only done in case the condition is true.